.github/workflows/release.yml

```diff
@@ -5,7 +5,27 @@ on:
     types: [prereleased]
 
 jobs:
-  release:
+  unit-test:
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Run Unit Test
+      run: make test
+
+    - name: Write result
+      if: always()
+      run: echo ${{ job.status }} > job-result.txt
+
+    - name: Upload job result for job slack-notify
+      if: always()
+      uses: actions/upload-artifact@v1
+      with:
+        name: result
+        path: job-result.txt
```

Comment:
This code is duplicated, but not support YAML's anchor and alias in GitHub Actions currently :(
https://github.community/t5/GitHub-Actions/Support-for-YAML-anchors/td-p/30336